src/ReduxStore/actions/currentCity.js

```diff
@@ -0,0 +1,20 @@
+import { currentURL } from '../../common/constants';
+export const PREFIX = 'CURRENT_CITY_MANAGER/';
+
+export const CURRENT_CITY_LOADING = `${PREFIX}CURRENT_CITY_LOADING`;
+export const CURRENT_CITY_LOADED = `${PREFIX}CURRENT_CITY_LOADED`;
+export const CURRENT_CITY_FAIL = `${PREFIX}CURRENT_CITY_FAIL`;
+export const SET_CURRENT_CITY_NAME = `${PREFIX}SET_CURRENT_CITY_NAME`;
+
+export function getCurrentCityData(id) {
+  return async function (dispatch, getState, dataServices) {
+    try {
+      dispatch({ type: CURRENT_CITY_LOADING });
+      const data = await dataServices.getData(`${currentURL}${id}`);
```

Comment:
in the getData method you're passing a slice of the url here while in the getCitiesData you're only passing the location and handling the rest of the url in the service. I'd stick to one of those approaches so that it's more consistent how the methods are supposed to be used

Reply:
Good call. I think is it no place here to use urls - move them to service at least. The onyl thing you better do here is 
`await dataServices.getData(id or params)`